Source/WebCore/contentextensions/NFAToDFA.cpp

```diff
@@ -275,7 +275,7 @@ struct DataConverterWithEpsilonClosure {
         for (unsigned nodeId : iterable) {
             result.add(nodeId);
             const UniqueNodeList& nodeClosure = nfaNodeclosures[nodeId];
-            result.add(nodeClosure.begin(), nodeClosure.end());
+            result.addAll(nodeClosure);
```

Comment:
No need for the local variable.
```suggestion
            result.addAll(nfaNodeclosures[nodeId]);
```